src/components/new-request/search.vue

```diff
@@ -103,11 +103,44 @@
       <!-- Change Name flow -->
       <template v-else-if="isChangeName">
         <BusinessLookupFetch />
-
+        <v-dialog  v-if="isSocietyDisabled" v-model="showSocietiesInfo"
```

Comment:
Please move this to its own component, same as the other dialogs in this project.

Reply:
Thanks.